app/models/null_identity.rb

```diff
@@ -1,3 +1,4 @@
+# Null object pattern substitute for ServiceProviderIdentity
```

Comment:
Should this be renamed to NullServiceProviderIdentity?

Reply:
I could go both ways....since there is no service provider... it's just a null identity?

How strongly do you feel?

Reply:
Not very, the comment makes it clear enough